packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -157,17 +156,19 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
                             ['ms-DatePicker-day--today ' + styles.dayIsToday]: day.isToday,
                             ['ms-DatePicker-day--highlighted ' + styles.dayIsHighlighted]: day.isSelected
                           }) }
-                        role='gridcell'
+                        role='link'
+                        href='#'
```

Comment:
we have assigned `role=link` which requires `href` to be present. check https://dequeuniversity.com/library/aria/date-pickers/sf-date-picker